noxfile.py

```diff
@@ -103,6 +103,10 @@ def unit_noextras(session):
 @nox.session(python=DEFAULT_PYTHON_VERSION)
 def pytype(session):
     """Run type checks."""
+    # An indirect dependecy attrs==21.1.0 breaks the check, and installing a less
+    # recent version avoids the error until a possibly better fix is found.
+    # https://github.com/googleapis/python-bigquery/issues/655
+    session.install("attrs==20.3.0")
```

Comment:
Shouldn't this also be set in setup.py? Presumably with `<=`?

Reply:
OK. I'd love to get a knowledge dump from you on `pytype`. :)

AFAIK, our code doesn't use `attr` directly.

It seems like you should be able to just tell `pytype` to ignore 3rd-party modules/packages, but I don' see a way to do that.  Apparently, `pytype` can use a configuration file, but they aren't documented AFAICT.

It seems it should be possible to exclude `attr` using `-x`, but I haven't figure out how. 

`pytype` is barely documented afaict.

Reply:
Yeah, that would also be my preference to silence that false positive without monkey-patching the dependency or by silencing all 3rd party modules. I think I'll create an issue to re-visit this workaround in the future.